recipes/resonance/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "resonance" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "af554059984027a1419a5359339c0ecc1543edd9bdf107b07f6ae6722c51f063" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python >=3.5
+    - setuptools
+  run:
+    - python >=3.5
```

Comment:
Python versions should be skipped using `build: skip` statements. Otherwise bad things happen. For eg: in circleci log, the package was compiled for 3.6 and then imported in 3.5.

Reply:
Ah. So a generic conda package shouldn't pin python versions in the recipe? meaning that python is treated as something different that any other given dependency?

Reply:
Yes. python needs to be treated differently to allow for building against different versions. Otherwise we would have to resort to pinning like we do for boost, openblas, etc.